src/main/java/org/openstreetmap/atlas/geography/atlas/items/complex/buildings/HeightConverter.java

```diff
@@ -8,8 +8,12 @@
 /**
  * Example use: Building Height
  *
+ * @deprecated use {@link org.openstreetmap.atlas.tags.HeightTag} {@code get()} or
+ *             {@link org.openstreetmap.atlas.tags.annotations.extraction.AltitudeExtractor}
+ *             instead.
  * @author matthieun
  */
+@Deprecated
```

Comment:
What would be the cost of just removing it in this PR?

Reply:
The only cost would be if there is something downstream that is using it. I am not aware of anything that is, but wanted to play it safe. If you think it is safe enough, I am happy to remove it.

Reply:
Ok, let's keep it then. We can then do a sweep later for all the deprecated annotations and remove the ones unused.